serial_server.py

```diff
@@ -54,27 +57,66 @@ class NoPortsAvailableError(Error):
     code = 3
 
 
+SerialDevice = collections.namedtuple('SerialDevice', ['name', 'devicepath'])
+
+
 class SerialServer(LabradServer):
     """Provides access to a computer's serial (COM) ports."""
     name = '%LABRADNODE% Serial Server'
 
     def initServer(self):
+        if sys.platform.startswith('win32'):
```

Comment:
not sure, but AFAICT the canonical way to check this is to use the `platform.system` function (https://docs.python.org/2/library/platform.html#platform.system):
```python
import platform
...
if platform.system() == 'Windows':
```